tests/system/providers/google/leveldb/example_leveldb.py

```diff
@@ -40,6 +45,19 @@
         command='put',
         key=b'another_key',
         value=b'another_value',
+        trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
What this operator is doing, do we need a trigger rule ALL_DONE here? It's usually used when we want to ensure that operator always runs (for example if operator removes some resource at the end of test)

Reply:
No, Updated the PR